nx-dev/feature-doc-viewer/src/lib/content.tsx

```diff
@@ -36,6 +36,9 @@ const components: any = (config: ComponentsConfig) => ({
       </code>
     );
   },
+  em({ children }) {
```

Comment:
I think I would remove this, I do not think we want to have all _italic_ string painted in green.

Reply:
That was the only tag that made sense to use for types. The alternative is to implement parsing of the markup to parse those to a React component. We can do that in a separate PR I guess.

Reply:
I wouldn't put time on this though. I think it would be better to change the way we generate content from yargs. Going to the `schema` direction as we do for executors and generators would enable us to use the same view and have a more reliant way to handle information.